runtests.sh

```diff
@@ -740,7 +740,7 @@ if [ $doNetTests = true ]
 then
     set +e  # disable exit on failure so that diagnostics can be given on failure
     echo "${separator}${blue}integration${noColor}"
-    for i in tests/*integration_*.py
```

Comment:
The logic in Line731 should also be updated regarding the change like this
[tests/test_cv2_dist.py → tests/config/test_cv2_dist.py](https://github.com/Project-MONAI/MONAI/pull/8231/files#diff-8b3495712e5106885db0d3f2e86ce8cfff6fd0c8847eb3b0e911b4298078ba90)

Reply:
Well observed.